ui/src/app/views/workspace/component.spec.ts

```diff
@@ -137,6 +138,25 @@ describe('WorkspaceComponent', () => {
 
   }));
 
+  it('displays correct notebook information', fakeAsync(() => {
+    let noteBookSize: number;
+    workspacePage.workspacesService.getNoteBookList(
+    workspacePage.workspaceNamespace,
+    workspacePage.workspaceId).subscribe((noteBookList) => {
+      noteBookSize = noteBookList.length;
+    });
+    tick();
+    expect(noteBookSize).toBe(1);
```

Comment:
This test doesn't seem meaningful. It is just asserting that the callback that you setup in the test above got called. Am I missing something?

Reply:
Yes the thought process behind this was that it is calling the right backend service